.jenkins/perf_test/compare_with_baseline.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import json
+import numpy
+from scipy import stats
+
+with open('../perf_test_numbers.json') as data_file:
+    data = json.load(data_file)
+
+test_name = sys.argv[1]
```

Comment:
I have never regretted putting a small `--help` text explaining what a program is and what it does at the top of my script. Maybe do it if `len(sys.argv) != 2`